cli/cmd/install.go

```diff
@@ -442,7 +442,7 @@ func (options *installOptions) buildValuesWithoutIdentity(configs *pb.All) (*ins
 		values.WebResources = &*defaultConstraints
 
 		values.IdentityResources = &resources{
-			CPU:    constraints{Request: "10m"},
+			CPU:    constraints{Request: "100m"},
 			Memory: constraints{Request: "10Mi"},
 		}
```

Comment:
Just curious, why isn't identity using `defaultConstraints`, like the rest of our go control plane components? Or put another way, should we also modify the default constraints CPU request to be 100m instead of 20m, and start using that here instead?

Reply:
yeah, we should probably update the defaults, but the identity controller maintains _no_ state, so it really doesn't need much in the way of memory reservations.